.travis.yml

```diff
@@ -13,8 +13,11 @@ jobs:
   include:
     # On PRs and branches, quickly run tests in debug mode.
     - if: type = pull_request
-      script: make test
+      script:
+        - make test
       rust: stable
+      before_cache:
+        - rm -rf target/debug/linkerd2-proxy target/release
```

Comment:
Is the culprit actually just the `cache: cargo` line above?

Reply:
we _want_ `cache: cargo`, since it enables caching of intermediate artifacts and dependencies and such. There are just parts of the `target` directory that aren't worth caching.